### ISSUE TITLE
Random Bug

### ISSUE THREAD

**Issue by bogointeractive** [Sunekaer/stoneBlock#703]
## Bug Report

Tinker Smelter not working correctly. IE I have it auto pulling out material to the casting bin. To put the items in I use AE2 export. I have a casting table as well for the off that does not make a block. I have a block that is in the casting bin. I also have the issues with AE2 and flux networks that seem to lose power. As I go into the crafting remote it says no network power and then it catches up 2 mins later. This is all connected together. It has been happening since I updated to the latest. I also have to where the client will Freeze and I have to use the Kill on the instance.
## Steps to Reproduce (for bugs)

1.
2.

## Logs

* Client/Server Log:
* Crash Log:

## Client Information

* Modpack Version: 1.0.31
* Server/LAN/Single Player: Single Player
* Optifine Installed: No
* Shaders Enabled: No

These are pictures showing that the casting basin is full but really not with the mouse over. And there is still things in the smeltery but nothing is coming out. 
![untitled1](https://user-images.githubusercontent.com/28167934/49477506-c388eb80-f7e2-11e8-9b03-3ddd55ac9b1f.png)

![untitled](https://user-images.githubusercontent.com/28167934/49477445-9d634b80-f7e2-11e8-8961-e31b4ecf0956.png)

**Comment by bogointeractive** [comment]
I also forgot to mention that when I break something it hovers and then flies up and down like it is hitting a lag spike or something. I am trying to get a screen capture of it. I know that I just used the pickaxe to break the casting bin and the glowstone and the transposer sat there in limbo for a good 30 secs or more.

**Comment by bogointeractive** [comment]
![untitled2](https://user-images.githubusercontent.com/28167934/49477894-ec5db080-f7e3-11e8-888d-861c5745884b.png)
Here is a screen grab showing the casting bin is full and hardened and it is not being sucked out by the AE2 import bus that is under it. This has worked perfectly until the update of 1.0.31.

**Comment by bogointeractive** [comment]
I also having issues with Extra Utilities 2 Generators and items going in and out. The project E condensers are not using the endio or item ducting to pull from them into the generators. I have gone back to 1.0.30 and it is working perfectly.

**Comment by Sunekaer** [comment]
None of the stuff in update 1.0.31 should have anything to do with this, hmm ill have to think about this one

**Comment by bogointeractive** [comment]
some of the tinkers stuff is still present in 1.0.30, I can not get anything out of the castin bin. also my RFTools, Flux powers are not staying active in powering. I have to use creative ones which I don't want to use but I have no choice.

**Comment by bogointeractive** [comment]
I have tried to redo it on my end and it still shows issues but I am deleting the smeltery because of its lag issues. I have tried it again in one of the new worlds to see if it still had the issue. Yes it still does in the mining world for some reason. I have tried to unclaim and reclaim the chunk. Still an issue as it shows the image like the above pictures show. So I am just going to do without the smeltery for now unless something comes up.
